css/properties/clip.json

```diff
@@ -53,12 +51,12 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "≤80"
+                "version_added": "1"
               },
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": "≤72"
+                "version_added": "3.5"
```

Comment:
Firefox 1.0 seems to have [already supported it](https://github.com/caugner/css-properties/commit/ddf4a9eb5a584f241c1874f599dfb0b16cdcdb29#diff-b63d72471087ee495d4f5c7e94b58bef200825135bdee1bdcddbfbd594b0c328R52):

```suggestion
                "version_added": "1"
```

Reply:
See Collector result: <img width="1086" alt="image" src="https://github.com/user-attachments/assets/35a33fd3-4205-4283-a86f-a79000ac4858" />

Reply:
We should probably fix the collector test then before merging this PR!

Reply:
How would you fix the collector test? Introduce a `isSame()` helper to replace the comparison [here](https://github.com/openwebdocs/mdn-bcd-collector/blob/fb9c2975ad152cf64b944da01b287b03643a955f/static/resources/harness.js#L659)?

Reply:
Opened https://github.com/openwebdocs/mdn-bcd-collector/issues/2243 to track this issue.